### PR TITLE
apm/python: fix env vars; remove django warnings

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
### Motivation

- The environment variable positioning was not accurate.
- The Django warning is no longer required.
- The supported python version listed was outdated.

### Preview link

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/tracing/setup/python.md [hunk 1]
```diff
@@ -51,26 +51,24 @@ When using **ddtrace-run**, the following [environment variable options][7] can
...
-| `DD_TAGS`                       |             | A list of default tags to be added to every span, profile, and runtime metric e.g. `layer:api,team:intake`.      |
-
-It is recommended to use `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` to set `env`, `service`, and `version` for your services. Refer to the [Unified Service Tagging][9] documentation for recommendations on how to configure these environment variables.
+| `DATADOG_TRACE_DEBUG`              | `false`     | Enable debug logging in the tracer.                                                                                                                                                                                                                                         |
+| `DATADOG_PATCH_MODULES`            |             | Override the modules patched for this application execution. It should follow this format: `DATADOG_PATCH_MODULES=module:patch,module:patch...`.                                                                                                                            |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:55]:
```suggestion
| `DATADOG_PATCH_MODULES`            |             | Override the modules patched for this application execution. It should follow the format: `DATADOG_PATCH_MODULES=module:patch,module:patch...`.                                                                                                                            |
```

File: content/en/tracing/setup/python.md [hunk 2]
```diff
@@ -51,26 +51,24 @@ When using **ddtrace-run**, the following [environment variable options][7] can
...
 
 | Environment Variable               | Default     | Description                                                                                                                                                                                                                                                                 |
 | ---------------------------------- | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `DD_ENV`                           |             | Set the application’s environment e.g. `prod`, `pre-prod`, `staging`. Learn more about [how to setup your environment][9].                                                                                                                                                  |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:61]:
```suggestion
| `DD_ENV`                           |             | Set the application’s environment, for example: `prod`, `pre-prod`, `staging`. Learn more about [how to setup your environment][9].                                                                                                                                                  |
```

File: content/en/tracing/setup/python.md [hunk 3]
```diff
@@ -51,26 +51,24 @@ When using **ddtrace-run**, the following [environment variable options][7] can
...
+| `DD_SERVICE`                       |             | The service name to be used for this application. The value is passed through when setting up middleware for web framework integrations (e.g. Pylons, Flask, Django). For tracing without a web integration, it is recommended that you [set the service name in code](#integrations).      |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:62]:
```suggestion
| `DD_SERVICE`                       |             | The service name to be used for this application. The value is passed through when setting up middleware for web framework integrations like Pylons, Flask, or Django. For tracing without a web integration, it is recommended that you [set the service name in code](#integrations).      |
```

File: content/en/tracing/setup/python.md [hunk 4]
```diff
@@ -51,26 +51,24 @@ When using **ddtrace-run**, the following [environment variable options][7] can
...
+| `DD_VERSION`                       |             | Set the application’s version e.g. `1.2.3`, `6c44da20`, `2020.02.13`.                                                                                                                                                                                                       |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:63]:
```suggestion
| `DD_VERSION`                       |             | Set the application’s version, for example: `1.2.3`, `6c44da20`, `2020.02.13`.                                                                                                                                                                                                       |
```

File: content/en/tracing/setup/python.md [hunk 5]
```diff
@@ -51,26 +51,24 @@ When using **ddtrace-run**, the following [environment variable options][7] can
...
+| `DD_TAGS`                          |             | A list of default tags to be added to every span, profile, and runtime metric e.g. `layer:api,team:intake`.      |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:64]:
```suggestion
| `DD_TAGS`                          |             | A list of default tags to be added to every span, profile, and runtime metric, for example: `layer:api,team:intake`.      |
```

File: content/en/tracing/setup/python.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ further_reading:
 ---
 ## Compatibilty Requirements
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:22]:
```suggestion
## Compatibility requirements
```

File: content/en/tracing/setup/python.md [hunk 7]
```diff
@@ -21,7 +21,7 @@ further_reading:
...
 
-Python versions `2.7` and `3.4` and onwards are supported.  For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
+Python versions `2.7` and `3.5` and onwards are supported.  For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:24]:
```suggestion
Python versions `2.7+` and `3.5+` are supported.  For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
```

File: content/en/tracing/setup/python.md [hunk 8]
```diff
@@ -51,26 +51,24 @@ When using **ddtrace-run**, the following [environment variable options][7] can
...
-| `DD_TRACE_ANALYTICS_ENABLED`       | `false`     | Enable App Analytics globally for [web integrations][13].                                                                                                                                                                                                                   |
+| `DATADOG_PRIORITY_SAMPLING`        | `true`      | Enable [Priority Sampling][10].                                                                                                                                                                                                                                              |
+| `DD_LOGS_INJECTION`                | `false`     | Enable [connecting logs and traces Injection][11].                                                                                                                                                                                                                           |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/python.md:70]:
```suggestion
| `DD_LOGS_INJECTION`                | `false`     | Enable [connecting logs and trace injection][11].                                                                                                                                                                                                                           |
```

**Comment by ruthnaebeck** [comment]
@Kyle-Verhoog - There are conflicts to be resolved

**Comment by Kyle-Verhoog** [comment]
@ruthnaebeck addressed!

**Pull request merged**